pkg/asset/manifests/operators.go

```diff
@@ -169,8 +169,9 @@ func (m *Manifests) generateBootKubeManifests(dependencies asset.Parents) []*ass
 	etcdEndpointHostnames := make([]string, *installConfig.Config.ControlPlane.Replicas+1)
 	for i := range etcdEndpointHostnames {
 		if i == 0 {
-			etcdEndpointHostnames[i] = "bootstrap"
-			i++
+			etcdEndpointHostnames[i] = "etcd-bootstrap"
+			etcdEndpointHostnames[i+1] = fmt.Sprintf("etcd-%d", i)
```

Comment:
`etcdEndpointHostnames[i]` gets overwritten on the next loop iteration. It looks like this line should be removed.

Is it intended that the etcd hosts are 1-indexed? It looks like this is trying to make them 0-indexed.

Reply:
@crawford we changed this in the later versions. etcd hosts are currently 0-indexed